.pre-commit-config.yaml

```diff
@@ -1,35 +1,24 @@
 repos:
-  - repo: https://github.com/pycqa/isort
-    rev: 5.12.0
+  - repo: https://github.com/charliermarsh/ruff-pre-commit
+    rev: "v0.0.254"
     hooks:
-      - id: isort
-        args: ["--profile", "black"]
+      - id: ruff
+        args: [--fix, --exit-non-zero-on-fix]
 
   - repo: https://github.com/psf/black
-    rev: 23.1.0
+    rev: "23.1.0"
     hooks:
       - id: black
-        args: ["--preview"]
 
-  - repo: https://github.com/pycqa/flake8
-    rev: 5.0.4
-    hooks:
-      - id: flake8
-
-  - repo: https://github.com/terrencepreilly/darglint
-    rev: v1.8.1
-    hooks:
-      - id: darglint
+  #- repo: https://github.com/pre-commit/mirrors-mypy
+  #  rev: "v1.1.1"
+  #  hooks:
+  #    - id: mypy
+  #      args: [--ignore-missing-imports]
+  #      additional_dependencies: ["types-requests", "types-PyYAML"]
 
-  - repo: local
-    hooks:
-      - id: pylint
-        name: pylint
-        entry: pylint
-        language: system
-        types: [python]
-        args: [
-            "-rn", # Only display messages
-            "-sn", # Don't display the score
-            "--ignore-paths=.github/*,docs/*",
-          ]
+  #- repo: https://github.com/adrienverge/yamllint.git
+  #  rev: v1.17.0
+  #  hooks:
+  #    - id: yamllint
+  #      args: ["-d", "{extends: relaxed, rules: {line-length: disable}}"]
```

Comment:
yaml and pylint the same?

Reply:
yamllint is linting yaml files only. I suggest moving to ruff for all the linting purposes and replace flake8, pylint, etc with it, I've had a good experience doing with with some other packages.